ngraph/frontend/paddlepaddle/src/op/logical_ops.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <node_context.hpp>
+
+#include "default_opset.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pdpd {
+namespace op {
+template <typename T>
+NamedOutputs logical_ops(const NodeContext& node) {
+    auto x = node.get_ng_input("X");
+    auto y = node.get_ng_input("Y");
+    return node.default_single_output_mapping({std::make_shared<T>(x, y)}, {"Out"});
+}
+
+NamedOutputs logical_and(const NodeContext& node_context) {
```

Comment:
Can we use separate file for each conversion? As we have in the nGraph for operations?

Reply:
Sorry @ilyachur, not sure why you think we need separate them to multiple files. Each logical op is an instance of the template, simple and straightforward. There is a similar situation for elementwise ops already. And there are also multiple reduce_XX ops as well. I prefer to clustering close ops in a single file. How do you think?